src/__tests__/App.spec.tsx

```diff
@@ -48,6 +49,7 @@ describe('App', () => {
 						id="learn"
 						renderer={({ params }) => <Learn guideName={params.guide} pageName={params.page} />}
 					/>
+					<Outlet key="dojoswag" id="dojoswag" renderer={() => <DojoSwag />} />
```

Comment:
Please add the outlet to the `pages` variable below, so that its renderer can be tested as well.
![image](https://user-images.githubusercontent.com/1388138/72535172-1e926c00-3846-11ea-942f-22af1c1ac424.png)

Reply:
Done.